app/models/event.rb

```diff
@@ -240,5 +241,11 @@ def user_cannot_be_inactive
 
     errors.add(:base, "You need an active subscription to capture events with your live API key")
   end
+
+  def property_cannot_be_all
+    return unless properties.keys.include?("All") || properties.keys.include?("all") 
```

Comment:
super weird, because I tried to do this like you did above, but if I put both statements in two return unless after each other, it's not working! why does it work above? where you did it for the api key validation?

Reply:
I think it would be good to make this more flexible so that we can more easily expand it in the future. For example, we can define the values we want to exlucde in an array, and then see if any of the values is included int the `properties.keys` array.

Here's a [StackOverflow link](https://stackoverflow.com/questions/25840356/array-include-multiple-values/25854899) that can help you with this.